Kconfig.zephyr

```diff
@@ -49,6 +49,9 @@ menu "Build and Link Features"
 
 menu "Linker Options"
 
+config FAIL_this
+	bool "test parsing"
+
```

Comment:
Let's not create noise for our users with this test code.

If this breaks in the future we will fix it again.

Reply:
Yes, this was included by mistake.

Removed.